### PR TITLE
Time Series Modeling with ARIMA in R

### PR DESCRIPTION
**Pull request by willyiamyu**

### CODE REVIEW

File: time_series_arima.Rmd [hunk 1]
```diff
@@ -0,0 +1,133 @@
+# Time Series Modeling with ARIMA in R
+
+William Yu
+```{r setup, include=FALSE}
+ knitr::opts_chunk$set(warning = FALSE, message = FALSE)
```
Comment (kmsingh-git) [inline time_series_arima.Rmd]: Hey, please remove this.

**Review by kmsingh-git — CHANGES_REQUESTED** [review]
Just one change in the .rmd file

File: time_series_arima.Rmd (hunk 1 above)
Comment (willyiamyu) [reply time_series_arima.Rmd]: Hi, is there any way I can edit this without commiting pushing and merging again? Can I edit the file directly on here?

Comment (willyiamyu) [reply time_series_arima.Rmd]: Oh nevermind, I think I got it.

**Review by kmsingh-git — APPROVED** [review]

**Comment by kmsingh-git** [comment]
Travis builds, looks good from my side

**Comment by bojum** [comment]
🎉 Thanks for your contribution! The branch has been merged!

**Pull request merged**